app/controllers/hyrax/dashboard/collections_controller_decorator.rb

```diff
@@ -19,21 +20,14 @@ def show
         super
       end
 
-      # OVERRIDE Hyrax v5.0.0 to add the ability to upload a collection thumbnail - START
```

Comment:
I moved this down since it shouldn't be getting used any more and didn't need as prominent of location in the module.